css/css-flexbox/percentage-descendant-of-anonymous-flex-item.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Flexbox Test: percentage heights in descendants of anonymous flex items</title>
+<link href="resources/flexbox.css" rel="stylesheet">
+<link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com">
+<link rel="match" href="reference/percentage-descendant-of-anonymous-flex-item-ref.html">
+<style>
+.flexbox {
+    width: 200px;
+    height: 200px;
+}
+</style>
+<!--
+    Both Blink and WebKit implement buttons as flexboxes, which in this case, wrap the child in an anonymous box.
+    Anonymous boxes are skipped when computing percentage heights but we need to ensure that their children with
+    percentage heights are properly sized.
+-->
```

Comment:
Nit: Might be nice to add a message for users about when the test i passing (like you should see a green rectangle).

Reply:
You mean inside that comment or in a visible (`<p>`...) element?

Reply:
Inside a <p> is usually nice in reftests.